pymc3/distributions/multivariate.py

```diff
@@ -1940,6 +1943,75 @@ def _distr_parameters_for_repr(self):
         return ["mu"]
 
 
+class CARRV(RandomVariable):
+    name = "car"
+    ndim_supp = 1
+    ndims_params = [1, 2, 1, 1]
```

Comment:
Are the last two dimensions really supposed to be vectors?  Below it states that they can be scalars.